components/footer/template.html

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
@uniyalabhishek template should only have the relevant tag(div, footer etc) not all the tags